yarn.lock

```diff
@@ -73,6 +73,46 @@
     uuid "^8.3.2"
```

Comment:
Don't think we need to update these dependencies. These are probably remnants of `setup-dev` run?

Reply:
This is in main, I guess we can target a separate PR in main for this